test/integration/connect/envoy/case-cfg-resolver-defaultsubset/setup.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+set -euo pipefail
+
+# wait for bootstrap to apply config entries
+wait_for_config_entry proxy-defaults global
+wait_for_config_entry service-resolver s2
+
+gen_envoy_bootstrap s1 19000
+gen_envoy_bootstrap s2 19001 # unused
```

Comment:
Why keep it?

Reply:
I wanted to keep around an instance of `service:s2` so that a misconfig with subset selection plumbing (losing the subset and just requesting plain `s2`) would have an instance to spill over incorrectly into to highlight the bug.